### ISSUE TITLE
COPR real time kernel stutters, any extra configuration needed?

### ISSUE THREAD

**Issue by msmafra** [CachyOS/linux-cachyos#199]
Hello there!

I decided trying the kernel-cachyos-rt 6.6.1 from the COPR, but I getting stutters in games and videos/stream. Audio quickly cracks the video freezes for a second or so and gets back to normal for a while. It also seems to have an effect on the networking. I saw spikes on the graphics while monitoring with Gnome's system monitor in the moments it happens.
Testing with kernel-cachyos 6.6.1 haven't not notice it yet.
Should've done something else after installing the RT kernel?

**Comment by ventureoo** [comment]
Can you please give the output of ``zcat "PREEMPT_RT" /proc/config.gz``?

**Comment by msmafra** [comment]
[cachyos.txt](https://github.com/CachyOS/linux-cachyos/files/13391084/cachyos.txt)

**Comment by msmafra** [comment]
It seems that is fixed on 6.6.1-cbrt4.0. Had no issues yesterday. Thanks! :)
```
❯ uname -a
Linux tars 6.6.1-cbrt4.0.fc39.x86_64 #1 SMP PREEMPT_RT Sat Nov 18 22:48:35 UTC 2023 x86_64 GNU/Linux

❯ zcat -a "PREEMPT_RT" /proc/config.gz | tee cachyos.txt
...

❯ grep -E "PREEMPT_RT" cachyos.txt
CONFIG_PREEMPT_RT=y

❯ grep -E "PREEMPT_RT" cachyos.txt -C 5
CONFIG_HAVE_PREEMPT_AUTO=y
# CONFIG_PREEMPT_NONE is not set
# CONFIG_PREEMPT_VOLUNTARY is not set
# CONFIG_PREEMPT is not set
# CONFIG_PREEMPT_AUTO is not set
CONFIG_PREEMPT_RT=y
CONFIG_PREEMPT_COUNT=y
CONFIG_PREEMPTION=y
CONFIG_SCHED_CORE=y

```

**Comment by ventureoo** [comment]
Yes, it was related to a previous issue https://github.com/CachyOS/linux-cachyos/issues/193.  COPR doesn't synchronize very quickly with changes in PKGBUILD, so that turned out to be the issue.
